src/Helper/GushQuestionHelper.php

```diff
@@ -20,10 +20,31 @@
 class GushQuestionHelper extends SymfonyQuestionHelper
 {
     /**
+     * @var int
+     */
+    private $attempts;
+
+    /**
+     * Set the maximum attempts when none is set.
+     *
+     * This should only be used for testing.
+     *
+     * @param int $attempts
+     */
+    public function setMaxAttempts($attempts)
```

Comment:
Doing this with a `TestQuestionHelper` introduced to many issues as helpers depending on the QuestionHelper also needed to be registered.